repos/system_upgrade/el7toel8/actors/checklocalnetworkscripts/actor.py

```diff
@@ -0,0 +1,28 @@
+import os
+import errno
+
+from leapp.actors import Actor
+from leapp.models import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+from leapp.libraries.actor.library import generate_report
+
+
+class LocalNetworkScriptsCheck(Actor):
+    """
+    Check for existence of network-scripts local scripts and warn user about possible
+    manual intervention requirements.
+    """
+
+    name = "local_network-scripts_check"
+    produces = (Report,)
+    tags = (IPUWorkflowTag, ChecksPhaseTag)
+
+    def process(self):
+        if (
+            os.path.isfile("/sbin/ifup-local")
+            or os.path.isfile("/sbin/ifup-pre-local")
+            or os.path.isfile("/sbin/ifdown-local")
+            or os.path.isfile("/sbin/ifdown-pre-local")
+        ):
+            generate_report()
```

Comment:
for the purpose of unit-tests, it's better to put the whole process functionality in the private library. e.g see the scandasd actor: https://github.com/oamg/leapp-repository/blob/b7ebfca50c3ad895d9893541f912b8197469d408/repos/system_upgrade/el7toel8/actors/scandasd/actor.py#L24

as well, please add some simple unit tests. look at the very same actor for inspiration how to mock `os.path.isfile` and parametrize decorator. Plus this one actor to see how to mock reporting in unit tests: https://github.com/oamg/leapp-repository/blob/b7ebfca50c3ad895d9893541f912b8197469d408/repos/system_upgrade/el7toel8/actors/checkcpu/tests/test_check_cpu_checkcpu.py#L44

Reply:
I included the whole process functionality in the private library. And unit tests were also added.